### ISSUE TITLE
`FlxSave` no longer works in mobile?

### ISSUE THREAD

**Issue by MAJigsaw77** [HaxeFlixel/flixel#2703]
Title

**Comment by Geokureli** [comment]
I know what you mean because someone else brought this up to me but in the future try to be more descriptive than "Title"

**Comment by MAJigsaw77** [comment]
So, I tried to use the migration API but it didn't work
```haxe
FlxG.save.bind('data');
```
After that I tried to make a debug build and this happened
![Screenshot_2022-12-12-14-38-17-915_com rozebud fpsplus](https://user-images.githubusercontent.com/77043862/209053210-08e7fc87-0bf9-4701-8952-4f291cfb62f4.jpg)

**Comment by MAJigsaw77** [comment]
I think I'll need to go back to flixel 4.11.0.

**Comment by Geokureli** [comment]
for now, that's a good idea. I'll let you know when I find a solution

**Comment by MAJigsaw77** [comment]
👍

**Comment by Geokureli** [comment]
what happens if you try on this branch?
https://github.com/Geokureli/flixel/tree/android_save

**Comment by MAJigsaw77** [comment]
There's no changes on `FlxSave` but maybe I can see the ploblem.
```haxe
var directory = lime.system.System.applicationStorageDirectory;
var path = haxe.io.Path.normalize('$directory/../../../$localPath') + "/";
```

In android and ios if you try to exit the application dif and to save something on it it can't because it needs root on android and jailbreak on ios

**Comment by Geokureli** [comment]
> There's no changes on `FlxSave` but maybe I can see the ploblem.
My bad, I thought I committed my changes to FlxSave. try it now
https://github.com/HaxeFlixel/flixel/compare/dev...Geokureli:flixel:android_save?expand=1

**Comment by MAJigsaw77** [comment]
I highly believe that this issue is on iOS aswell.

**Comment by Geokureli** [comment]
I plan to include ios in the fix, just want to get this tested first

**Comment by Geokureli** [comment]
@MAJigsaw77 were you able to test it?

**Comment by MAJigsaw77** [comment]
I tested, still doesn't work

**Comment by Geokureli** [comment]
does it work if you use 4.11?

**Comment by MAJigsaw77** [comment]
I used the latest for test

**Comment by Geokureli** [comment]
> I used the latest for test

I don't know what you mean. if you run `haxelib set flixel 4.11.0` does saving work on mobile?

**Comment by MAJigsaw77** [comment]
Yea but without the file you changed

**Comment by MAJigsaw77** [comment]
@Geokureli what to do?

**Comment by Geokureli** [comment]
I said above that you should use flixel 4.11 until a fix is found. This is difficult for me because I don't build for android and December in general is a busy time for me, so i havent had a chance to fiddle. You'll need to either wait until i can start looking into this (maybe a couple weeks) or you can dog around yourself and see where i should i be saving on android

**Comment by MAJigsaw77** [comment]
Idk

**Comment by Geokureli** [comment]
You don't know what? I didn't ask a question.

**Comment by MAJigsaw77** [comment]
At the end you asked a question

**Comment by Geokureli** [comment]
No I said what you should look into if you want this solved faster

**Comment by MAJigsaw77** [comment]
![Screenshot_2023-01-03-02-58-31-781-edit_com android chrome](https://user-images.githubusercontent.com/77043862/210288268-13aa4259-5538-4932-bb49-940e19a2e95f.jpg)

**Comment by Geokureli** [comment]
And if you read a little further back, you'll see i wasn't asking for an answer i was telling you what the work involves

> you can dig around yourself and see where i should i be saving on android, and why my fix branch doesnt save there

**Comment by MAJigsaw77** [comment]
Ok

**Comment by Geokureli** [comment]
this is fixes in 5.2.1
